packages/elvis-illustrations/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "type": "module",
   "exports": {
+    "./CHANGELOG.json": "./CHANGELOG.json",
```

Comment:
Is this something we need to expose through our NPM package? 🙂 I don't find it very common to export changelogs in NPM packages.